server/modules/record/service/update/thread/recordUpdateThread.js

```diff
@@ -53,12 +58,19 @@ class RecordUpdateThread extends Thread {
       this.postMessage({ type, content })
   }
 
-  async handleNodesUpdated (updatedNodes) {
+  async handleNodesUpdated (updatedNodes, t) {
     this._postMessage(WebSocketEvents.nodesUpdate, updatedNodes)
```

Comment:
why are you passing the t parameter here?

Reply:
it's not required, right